README.md

```diff
@@ -1 +1,50 @@
 # HangOps Code of Conduct
+
+To continue growing a safe, welcoming, and fair community, all HangOps members are required to comply with the following code of conduct. HangOps admins are responsible for ensuring that all community members follow this code within HangOps spaces.
+
+## The Short Version
+
+Be respectful of other people; respectfully ask people to stop if you are bothered; and if you can’t resolve an issue contact the admins. If you are unable to follow our Community Code of Conduct problem, it will be apparent and you may be asked to leave HangOps.
```

Comment:
Thanks @jordansissel. Great feedback. I've updated each of the sections you noted but I haven't yet defined respect. I agree with you that we should probably be clear about what we mean by that word since it's so foundational to this CoC.